### DISCUSSION TITLE
I get the latest image not running under debian buster.

### DISCUSSION THREAD

**Discussion by schwidom** [docker-library/postgres#1185] [General]
https://stackoverflow.com/questions/77811138/the-official-docker-image-postgres-is-not-running-under-debian-buster

**Comment by tianon** [comment]
The most likely cause is seccomp -- try updating `libseccomp2`, Docker, and runc on your host (especially to newer versions than are available in Debian Buster, which is very close to EOL; 2024-06-30).
